Assets/Scripts/ArenasParametersSideChannel.cs

```diff
@@ -1,47 +1,43 @@
 using Unity.MLAgents.SideChannels;
 using System;
 using ArenasParameters;
+using UnityEngine;
 
 /// <summary>
 /// This class is used to communicate the environment configurations to the Unity.
 /// </summary>
 public class ArenasParametersSideChannel : SideChannel
 {
-	/// <summary>
-	/// Initializes a new instance of the ArenasParametersSideChannel class.
-	/// </summary>
-	public ArenasParametersSideChannel()
-	{
-		ChannelId = new Guid("9c36c837-cad5-498a-b675-bc19c9370072");
-	}
+    /// <summary>
```

Comment:
This appears to be an indentation switch; do we have a standard for this package?

nit: these sorts of changes are worth looking out for before doing commits - they make reviewing particularly difficult as every line is highlighted as changed

Reply:
Yeah I recently started to code on my Mac (on windows as well). I think they are using diff formatting tools so I need to look into that. Thanks for the spot :)